src/main/scala/cromwell/util/Encryption.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.util
+
+import java.security.SecureRandom
+import javax.crypto.Cipher
+import javax.crypto.spec.{IvParameterSpec, SecretKeySpec}
+
+import org.apache.commons.codec.binary.Base64
+
+import scala.util.{Failure, Success, Try}
+
+case object Aes256Cbc {
+
+  val encryption = "AES"
+  val blockSize = 128
+  val keySize = 256
+  val paddingMode = "PKCS5Padding"
+  val cipherMode = "CBC"
+
+  val ranGen = new SecureRandom()
+
+  final def init(mode: Int, secretKey: Array[Byte], iv: Array[Byte]) = {
```

Comment:
So to walk through my mental processes which led to the first comment, it went like this ...

1) I see "iv" all over the place w/o really looking at what was going on.
2) I say "WTF is an iv?"
3) I realize it's all encryption-y stuff and say "Oh, initialization vector"

I don't think it's a huge deal either way as the delta-t between steps 2 & 3 weren't large (and I was heavily distracted and on my phone IIRC) so might be best to jsut ignore this whole thing